src/v2/Components/Authentication/FormSwitcher.tsx

```diff
@@ -181,13 +171,13 @@ export class FormSwitcher extends React.Component<FormSwitcherProps, State> {
     let Form: FormComponentType
     switch (this.state.type) {
       case ModalType.login:
-        Form = isMobile ? MobileLoginForm : LoginForm
+        Form = LoginForm
         break
       case ModalType.signup:
-        Form = isMobile ? MobileSignUpForm : SignUpForm
+        Form = SignUpForm
         break
       case ModalType.forgot:
-        Form = isMobile ? MobileForgotPasswordForm : ForgotPasswordForm
+        Form = ForgotPasswordForm
```

Comment:
**suggestion [if-easy]:** extract map

I have a preference to see a construct like this done with a map, something like this:

```ts
const modalTypeMap = {
  ModalType.login: LoginForm,
  ModalType.signup: SignUpForm,
  ModalType.forgot: ForgotPasswordForm
}

// ...

const Form = modalTypeMap[this.state.type]
```

What do you think?

Note: unless I'm off here, I believe we have a `default` that returns `null` but then in the rendering we don't guard against it? Maybe it doesn't matter in reality, but seems like a case that's not being handled and I realize that covering this is not strictly speaking a part of this PR. Just wanted to point out something I noticed while reading this file!!

*format via [Conventional Comments](https://conventionalcomments.org/)*

Reply:
Agreed that that's better! Is it all right if I leave that as a followup item for a separate auth code refactor? Other parts of the code would benefit from this.